content/23.vaccines-novel.md

```diff
@@ -360,75 +384,104 @@ In the vaccines developed under this approach, the mRNA coding for a stabilized
 Notably, immunocompromised individuals such as cancer patients seem to elicit a sufficient protective immune response against the Omicron variant when they have been boosted with a third dose of either mRNA vaccine, albeit a blunted response [@doi:10.1016/j.ccell.2021.12.014].
 While antibody titers do correlate with protection [@doi:10.1126/science.abm3425; @doi:10.1016/j.vaccine.2021.12.006; @doi:10.1016/j.vaccine.2021.05.063; @doi:10.1101/2021.05.21.21257572; @doi:10.1038/s41423-020-00588-2], they are not the only mechanisms of immune protection; for example, T cell and non-neutralizing antibody responses may be unaffected or less affected by the new VOC and they warrant further investigation.
 
-In countries such as Israel, a fourth dose of mRNA vaccines have been introduced in response to the Omicron variant and an initial study in healthcare workers show that the additional immunization is safe and immunogenic with antibody titers restored to peak-third dose titers.
-No severe illness was reported in the cohort studied (274 versus 426 age-matched controls), and vaccine efficacy against infection was reported at 30% for BNT162b2 and 11% for mRNA-1273 [@doi:10.1101/2022.02.15.22270948].
-Low efficacy against infection is not surprising considering the vaccines are intended to prevent against severe disease, hospitalization and death rather than infection.
 
+<!--Move to evolution
 Vaccine efficacy is not the only pharmacological intervention affected by VOC.
 Some existing therapeutics, including monoclonal antibody treatments like Bamlanivimab (AbCellera Biologics/ Eli Lilly), were ineffective against the Omicron variant.
 Indeed, only Sotrovimab (Vir Biotechnology/GSK) and Tixagevimab (AstraZeneca) to a much lesser extent could effectively neutralize the omicron variant out of 7 tested monoclonal antibodies [@doi:10.1101/2021.12.14.21267772], which has been verified by others [@doi:10.1101/2021.12.12.472269].
 The antigenic shift of the Omicron variant does raise concerns for future VOC and what effects they may have on future vaccines and therapeutics.
+-->
 
-**Serological Response/Boosters:**
+The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
+Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
+To adapt, booster shots are now recommended in many places, and vaccines in the near future will likely account for multiple variants and strains of SARS-CoV-2 [@doi:10.1038/s41591-021-01230-y].
 
-**Variants:**
+### Booster Doses
 
-### Special Populations
+Due to waning immunity and viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
```

Comment:
```suggestion
Due to waning neutralizing anitbodies and viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
```

Reply:
The immunity thing is debatable, what we really are talking about neutralizing antibodies as a measure of immunity. Both that negates the possible immune protection offered by other cells.